Lib/importlib/_bootstrap_external.py

```diff
@@ -1508,6 +1508,7 @@ def find_spec(cls, fullname, path=None, target=None):
                 # can create the namespace package.
                 spec.origin = None
                 spec.submodule_search_locations = _NamespacePath(fullname, namespace_path, cls._get_spec)
+                spec.loader = NamespaceLoader(fullname, namespace_path, cls._get_spec)
```

Comment:
(Nothing to do here.)

I just noticed that PathEntryFinder.find_spec() is expected to return a spec with `loader` set to None, rather than NamespaceLoader (which is mostly fine).  See #92896.

Reply:
Yep, that was the previous protocol, which we are changing now.